minkindr/include/kindr/minimal/implementation/rotation-quaternion-inl.h

```diff
@@ -33,8 +33,10 @@ namespace minimal {
 
 template <typename Scalar>
 struct EPS {
-  static constexpr Scalar value();
-  static constexpr Scalar normalization_value();
+  static constexpr Scalar value() { return static_cast<Scalar>(1.0e-5); }
+  static constexpr Scalar normalization_value() {
+    return static_cast<Scalar>(1.0e-4);
```

Comment:
I copied these values from below. Are those values set from experience. It does not seem clear to me how to choose them. In fact, for the double template specialization I would have expected something like std::numeric_limits<double>::epsilon() as value.